File: Core Java/oopjava/Day03/Demo02/src/com/sunbeam/shapes/Rectangle.java
package com.sunbeam.shapes;

import java.util.Scanner;

public class Rectangle {
	int length;
	int breadth;
	
	public void acceptData() {
		Scanner sc = new Scanner(System.in);
		
		System.out.print("Enter length - ");
		length = sc.nextInt();
		
		System.out.print("Enter breadth - ");
		breadth = sc.nextInt();
	}
	
	public void displayData() {
		System.out.println("Area of Rectangle - "+ (length*breadth));
	}
}

File: cpp/Day06/demo12/src/rectangle.cpp
#include "../include/rectangle.h"

void Rectangle::acceptData()
{
    cout << "Enter the length = ";
    cin >> this->length;
    cout << "Enter the breadth = ";
    cin >> this->breadth;
}

void Rectangle::calculateArea()
{
    cout << "Area of rectangle = " << this->length * this->breadth << endl;
}